samples/extensions/vertex_dynamic_state/vertex_dynamic_state.cpp

```diff
@@ -371,12 +335,38 @@ void vertex_dynamic_state::build_command_buffers()
 		render_pass_begin_info.framebuffer              = framebuffers[i];
 		render_pass_begin_info.renderArea.extent.width  = width;
 		render_pass_begin_info.renderArea.extent.height = height;
-		render_pass_begin_info.clearValueCount          = 3;
+		render_pass_begin_info.clearValueCount          = clear_values.size();
```

Comment:
You probably should `static_cast<uint32_t>(clear_values.size())`

Reply:
Yes, you have a right.